client/accounting.go

```diff
@@ -61,15 +76,15 @@ func (c *Client) BalanceGet(ctx context.Context, prm PrmBalanceGet) (accounting.
 	// init call context
 
 	var (
-		cc  contextCall
 		res accounting.Decimal
+		cc  contextCall
```

Comment:
Not a useful change (moving var definition).

Reply:
Change removed